python/tvm/autotvm/tuner/tuner.py

```diff
@@ -86,8 +87,7 @@ def update(self, inputs, results):
             result for measurement
         """
 
-
-    def tune(self, n_trial, measure_option, early_stopping=None, callbacks=()):
+    def tune(self, n_trial, measure_option, early_stopping=None, callbacks=(), allow_skipping=True):
```

Comment:
I think the default value of `allow_skipping` should be `False`, which means users have to intentionally skip tasks.